packages/tokens/dist/products/css/themed-tokens/with-css-selectors--advanced/tokens.css

```diff
@@ -0,0 +1,538 @@
+  .hds-theme-system {
+    --token-color-palette-neutral-0: #ffffff;
+    --token-color-foreground-primary: #161616;
+    --token-typography-font-weight-regular: 400;
+    --token-typography-font-weight-medium: 400;
+    --token-typography-font-weight-semibold: 600;
+    --token-typography-font-weight-bold: 600;
+  }
+}
+
+@media (prefers-color-scheme: dark) {
+  .hds-theme-system {
+    --token-color-palette-neutral-0: #ffffff;
+    --token-color-foreground-primary: #f4f4f4;
+    --token-typography-font-weight-regular: 400;
+    --token-typography-font-weight-medium: 400;
+    --token-typography-font-weight-semibold: 600;
+    --token-typography-font-weight-bold: 600;
+  }
+}
+
+.hds-theme-light,
+.hds-mode-cds-g0 {
+  --token-color-palette-neutral-0: #ffffff;
+  --token-color-foreground-primary: #161616;
+  --token-typography-font-weight-regular: 400;
+  --token-typography-font-weight-medium: 400;
+  --token-typography-font-weight-semibold: 600;
+  --token-typography-font-weight-bold: 600;
+}
+
+.hds-theme-dark,
+.hds-mode-cds-g100 {
+  --token-color-palette-neutral-0: #ffffff;
+  --token-color-foreground-primary: #f4f4f4;
+  --token-typography-font-weight-regular: 400;
+  --token-typography-font-weight-medium: 400;
+  --token-typography-font-weight-semibold: 600;
+  --token-typography-font-weight-bold: 600;
+}
+
+.hds-mode-cds-g10 {
+  --token-color-palette-neutral-0: #ffffff;
+  --token-color-foreground-primary: #161616;
+  --token-typography-font-weight-regular: 400;
+  --token-typography-font-weight-medium: 400;
+  --token-typography-font-weight-semibold: 600;
+  --token-typography-font-weight-bold: 600;
+}
+
+.hds-mode-cds-g90 {
+  --token-color-palette-neutral-0: #ffffff;
+  --token-color-foreground-primary: #f4f4f4;
+  --token-typography-font-weight-regular: 400;
+  --token-typography-font-weight-medium: 400;
+  --token-typography-font-weight-semibold: 600;
+  --token-typography-font-weight-bold: 600;
+}
+
+/*
+
+WHY ARE ALL "COMPUTED" CSS VARIABLES REDECLARED IN EACH THEME SCOPE?
+
+CSS custom properties (variables) inherit from parent to child, but when a variable is defined
+as a calculation or alias of another variable (e.g., `--alias-value: var(--source-value)`, its value is fixed
+at the scope where it is declared—using whatever is visible at that level in the DOM (eg. `:root`).
+
+This block of "common/shared" CSS variables contains a lot of these aliases, so if we only declare these
+"computed" variables at `:root` level, their values do not update in themed blocks that override
+the "source" variables (e.g., `--source-value`) at a lower scope.
+This results in _stale_ variables, where the computed value is always stuck on the `:root` value,
+even inside a themed block where the source variable is assigned.
+
+To make sure every themed block/scoped context (eg. `.hds-theme-light`) correctly computes the aliased values,
+we redeclare each "computed" variable in every theme scope where its "source" variable is overridden.
```

Comment:
Oh, I guess this explains why my idea for defining variables for the various standardized font weights wouldn't work...